# Generated by Django 2.2.16 on 2022-11-18 08:22

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        ('users', '0026_auto_20220809_1223'),
    ]

    operations = [
        migrations.CreateModel(
            name='UserLogErrors',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('created', models.DateTimeField(auto_now_add=True, verbose_name='Время создания')),
                ('updated', models.DateTimeField(auto_now=True, verbose_name='Время изменения')),
                ('text', models.TextField(blank=True, null=True, verbose_name='Текст ошибки')),
                ('user', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='user_log_errors', to=settings.AUTH_USER_MODEL, verbose_name='Пользователь')),
            ],
            options={
                'verbose_name': 'User Log Errors',
                'verbose_name_plural': 'User Log Errors',
                'db_table': 'user_log_errors',
                'ordering': ['-created'],
            },
        ),
    ]
